### PR TITLE
ENH build older clang with new tapi

### PR DESCRIPTION
**Pull request by beckermr**
**Please merge to a branch**

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by beckermr** [comment]
@conda-forge-admin rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -19,7 +19,7 @@ source:
       - patches/530-ld64-add-conda-specific-env-vars-to-modify-lib-search-paths.patch
 
 build:
-  number: 16
+  number: 20
```
Comment (isuruf) [inline recipe/meta.yaml]:
```suggestion
  number: 17
```

**Comment by beckermr** [comment]
I suspect the older llvm won't work on osx-arm64. Shall we remove those builds or simply let them fail?

**Comment by isuruf** [comment]
It's okay to fail

**Comment by beckermr** [comment]
Alright. Then this is ready to merge @isuruf!

**Review by isuruf — APPROVED** [review]

**Comment by beckermr** [comment]
Shall I make a clang10 branch and merge to there?

**Comment by isuruf** [comment]
sure.

**Comment by isuruf** [comment]
can you send a PR to update the build number to 18 to master?

**Comment by beckermr** [comment]
See #32

**Pull request merged**